priv/templates/resource/controller_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule <%= module %>ControllerTest do
+  use <%= base %>.ConnCase
+
+  @valid_params [<%= singular %>: <%= sample_params %>]
+
+  test "GET /<%= plural %>" do
+    conn = get conn(), <%= singular %>_path(conn, :index)
+    assert conn.resp_body =~ "Listing <%= plural %>"
```

Comment:
This can now be `html_response(conn, 200) =~ "Listing <%= plural %>"`. Don't worry, I have changed it already but you will want to use the `html_response` stuff for the failure scenarios in create/update. :)